GPU/GPUTracking/Global/GPUChain.h

```diff
@@ -70,6 +70,7 @@ class GPUChain
   GPUReconstruction::RecoStepField GetRecoStepsGPU() const { return mRec->GetRecoStepsGPU(); }
   GPUReconstruction::InOutTypeField GetRecoStepsInputs() const { return mRec->GetRecoStepsInputs(); }
   GPUReconstruction::InOutTypeField GetRecoStepsOutputs() const { return mRec->GetRecoStepsOutputs(); }
+  inline GPUConstantMem* processorsShadow() { return mRec->mProcessorsShadow; }
```

Comment:
Why do you need this function to be public? As far as I can tell, it is only used by GPUChainITS, which derives from GPUChain.

In general, if needed, I have no problem with makeing it public, but then I'd actually make the whole block of functions public.

Reply:
You ar eright, must be a remnant of some kind of test of mine.